cmd/kubelet/app/server.go

```diff
@@ -500,11 +500,11 @@ func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) (cpuset.
 
 	topo, err := topology.Discover(machineInfo)
 	if err != nil {
-		return emptyCPUSet, fmt.Errorf("unable to discover CPU topology info: %s", err)
+		return emptyCPUSet, fmt.Errorf("unable to discover CPU topology info: %w", err)
```

Comment:
Accroding to the descriptions in the https://golang.org/pkg/fmt/. The returned error will implement an Unwrap method if we using %w instead of %s.  In other word ,we should call unwrap method first if we want to output error using Printf. Are you sure?

Reply:
Sorry for not look inside the `fmt` package. should I close it?